### ISSUE TITLE
Python api for map statistics (hazelcast cluster)

### ISSUE THREAD

**Issue by grafana2018** [hazelcast/hazelcast#13175]
Hi,

As per below document we use to get map statistics in java :
http://docs.hazelcast.org/docs/latest-development/manual/html/Management/Getting_Member_Statistics/Map_Statistics.html#page_Map+Statistics

We are implementing now the Python clien t for hazelcast cluster but not able find api for map statistics in python, please let us know is there anything lke that to get map statistics in pyhton by python-client.

Java --> LocalMapStats mapStatistics = customers.getLocalMapStats;
Python ----> ???

My code:
-----------
import hazelcast, logging
config = hazelcast.ClientConfig()
config.network_config.addresses.append('localhost:5701')                                    
logging.basicConfig()
logging.getLogger().setLevel(logging.INFO)
client = hazelcast.HazelcastClient(config)
my_map = client.get_map("map-name").blocking()
my_map.put("key_1", "value_1")
value = my_map.get("key_1")
print("Get val :"+value)
print("map.contains_key", my_map.contains_key("key"))
print("map.get", my_map.get("key"))
print("map.size", my_map.size())
print("map.putcount", my_map.get("putCount"))
client.shutdown()

Regards,
Sreenath

**Comment by grafana2018** [comment]
Thanks for the reply, Is this available in REST API's?

**Comment by vbekiaris** [comment]
No, I'm afraid map statistics are not available via Hazelcast members' REST API.
